examples/hosts/iframe-host/src/containerProxy.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as Comlink from "comlink";
+import { IContainer } from "@fluidframework/container-definitions";
+import { IRequest, IResponse } from "@fluidframework/core-interfaces";
+import { IQuorum } from "@fluidframework/protocol-definitions";
+
+export interface IContainerProxy {
+    attach(request: IRequest): Promise<void>;
+    request(request: IRequest): Promise<IResponse>;
+    getQuorum(): Promise<IQuorum>;
+    on(event: string, listener: (...args: any[]) => void): Promise<void>;
+    once(event: string, listener: (...args: any[]) => void): Promise<void>;
```

Comment:
i'm not sure we need to proxy these events. are they needed? we should be minimal here, and only handle what we need